src/index.ts

```diff
@@ -74,7 +74,8 @@
  * @property {constructor} SubscriberClient
  *     Reference to {@link v1.SubscriberClient}.
  */
-module.exports.v1 = require('./v1');
+import v1 = require('./v1');
+export {v1};
```

Comment:
Is this change testable?

Reply:
I'm sure it is, but we've never made a point to test exports in any of the clients. Although I could see some value in testing exports that have no types (like this). @JustinBeckwith @bcoe WDYT?

Reply:
Just a simple `assert.ok(spanner.v1)` would be nice :)

Reply:
Probably was heavy handed of me, but I've renamed the old `index.ts` test file to be `pubsub.ts` to match the actual source code we're testing. Now the `index.ts` test file just includes the export tests. The overall diff will be annoying to read, but I broke it up into logical commits if you want to review commit by commit.